src/systems/eu/implementations/react/packages/site-header/SiteHeader.jsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Icon from '@ecl/eu-react-component-icon/Icon';
+import Logo from '@ecl/eu-react-component-logo/Logo';
+import SearchForm from '@ecl/eu-react-component-search-form/SearchForm';
+
+const SiteHeader = ({
+  logo,
+  languageSelector,
+  searchForm,
+  className,
+  ...props
+}) => {
+  const classNames = classnames(className, {
+    'ecl-site-header': true,
+  });
+
+  return (
+    <div className={classNames} {...props}>
```

Comment:
why not use `<header>`? and `role="banner"`?

https://www.w3.org/TR/wai-aria-practices-1.1/examples/landmarks/banner.html
https://www.scottohara.me/blog/2018/03/03/landmarks.html

Reply:
`role="banner"` is reported as an error by a11y, but indeed it should be header and not div
![image](https://user-images.githubusercontent.com/25579221/45415359-c5c52900-b67d-11e8-8dd5-c6765a6bcb3e.png)